bench/demo.fpcore

```diff
@@ -40,3 +40,10 @@
  :name "jeff quadratic root 2"
  (let ((d (sqrt (- (* b b) (* (* 4 a) c)))))
    (if (>= b 0) (/ (* 2 c) (- (- b) d)) (/ (+ (- b) d) (* 2 a)))))
+
+(FPCore (p)
+  :herbie-target
```

Comment:
```suggestion
  :alt
```
No way for you to know this but nowadays we prefer `:alt` over `:herbie-target`, which I'm not sure is even supported.

Reply:
Also was there any precondition on the original user expression? Those are usually worth including, because they can clarify what the user actually expected to work.